app/renderer/reducers/frameReducer.js

```diff
@@ -112,26 +115,28 @@ const frameReducer = (state, action, immutableAction) => {
       }
 
       // TODO fix race condition in Muon more info in #9000
-      const title = immutableAction.getIn(['tabValue', 'title'])
+      const title = tab.get('title')
       if (title != null) {
         state = state.setIn(['frames', index, 'title'], title)
       }
 
       // TODO fix race condition in Muon more info in #9000
```

Comment:
I believe the race condition no longer exists, but `active` should be changed to check `changeInfo.active` if `changeInfo` is not empty and if `changeInfo` is empty (no attributes at all) it should check `tab`